packages/frontend/component/src/ui/input/style.css.ts

```diff
@@ -74,6 +74,11 @@ export const input = style({
     '&::placeholder': {
       color: 'var(--affine-placeholder-color)',
     },
+    '&:autofill, &:-webkit-autofill, &:-internal-autofill-selected, &:-webkit-autofill:hover, &:-webkit-autofill:focus, &:-webkit-autofill:active':
+      {
+        backgroundColor: 'var(--affine-white-10) !important',
```

Comment:
why use important ? write some comments to record

Reply:
The user agent style sheets of many browsers use !important in their :-webkit-autofill style declarations

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill